daffodil-runtime1/src/main/scala/org/apache/daffodil/dpath/NodeInfo.scala

```diff
@@ -253,16 +258,16 @@ object NodeInfo extends Enum {
    * hierarchy. We have it as a parent of both complex type and simple type since
    * both of them can be nillable.
    */
-  protected sealed trait NillableKind extends AnyType.Kind
-  case object Nillable extends TypeNode(AnyType, List(Complex, AnySimpleType)) with NillableKind {
-    type Kind = NillableKind
-  }
+  //  protected sealed trait NillableKind extends AnyType.Kind
```

Comment:
remove commented code - several of these type taxonomy nodes have been removed (2 or 3)